gulpfile.js

```diff
@@ -34,7 +34,9 @@ alphavilleBuildTools(gulp, {
 		{
 			id: 'ml-index',
 			standalone: 'mlIndexBundle',
+			js: './assets/js/marketLive.js',
 			sass: './assets/scss/ml-index.scss',
+			'build-js': 'ml-Index.js',
```

Comment:
is that capital "I" in Index intentional?

Reply:
Oh no. It is not. I will change it. Thank you.